arangodb-net-standard/GraphApi/GraphApiClient.cs

```diff
@@ -39,5 +40,30 @@ public async Task<GetGraphsResponse> GetGraphs()
                 throw await GetApiErrorException(response);
             }
         }
+
+        /// <summary>
+        /// Delete graph
```

Comment:
What about providing more information using the summary found in the ArangoDB docs?

I propose:

```
Deletes an existing graph object by name.
Optionally all collections not used by other graphs can be deleted as well, using <see cref="DeleteGraphQuery"></see>.
```